app/src/main/java/com/academy/ramon/marvelcomicviewer/MainActivity.java

```diff
@@ -49,11 +54,13 @@ public void getHeroesList(){
             public void onResponse(Call<HeroesResponse> call, Response<HeroesResponse> response) {
                 HeroesResponse heroesResponse;
                 heroesResponse = response.body();
-                rvHeroes.setAdapter(new HeroAdapter(heroesResponse.getItems(),rvHeroes.getContext()));
-                heroes = heroesResponse.getItems();
-                Log.d("hey there", response.body().toString());
-            }
+                heroes = heroesResponse.getData().getResults();
+                HeroAdapter adapter = new HeroAdapter(heroes, rvHeroes.getContext());
+                rvHeroes.setAdapter(adapter);
+                rvHeroes.setLayoutManager(new LinearLayoutManager(rvHeroes.getContext()));
```

Comment:
I dont know why this doesnt seem to be applicable in this case, hence I had to provide the view's contexts.